### DISCUSSION TITLE
Port requirements for Postal V3

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2876] [Help with using Postal] [answered]
Hello,

With the remove of RabbitMQ in V3 I just wanted to confirm the ports that need to be open to the internet.  Please confirm.
PORTS: 25, 443, 80, 53, 8080

**Comment by adamcooke** [comment, marked as answer]
## Incoming

* `TCP:80` and `TCP:443` for the web interface (and tracking)
* `TCP:25` for the SMTP server

## Outgoing

* `TCP:25` for sending e-mails to other SMTP servers
* `TCP:80` and `TCP:443` for sending e-mails to HTTP endpoints and webhooks
* `TCP:53` and `UDP:53` for DNS resolution
* `TCP:3306` for the MariaDB database if it isn't on the same host as the Postal application

If you have SMTP or HTTP endpoints which use other ports you will also need to allow these.

**Comment by willpower232** [comment]
You don't normally need to open 53 to the internet, it probably gives you a bad time unless you are hosting your own DNS.

8080 doesn't sound particularly related to Postal.
